lib/network_utils.ml

```diff
@@ -109,7 +109,12 @@ module Sysfs = struct
 				(fun () -> close_in inchan)
 		with
 		| End_of_file -> ""
-		| exn -> error "%s" (Printexc.to_string exn); raise (Read_error file)
+		(* Match the exception when the device state if off *)
+		| Sys_error("Invalid argument") -> raise (Read_error file)
```

Comment:
This is introducing a new warning:
```
Warning 52: Code should not depend on the actual values of
this constructor's arguments. They are only for information
and may change in future versions. (See manual section 8.5)
```
but I don't think that there is a nicer way of dealing with it at the moment.
The workaround would be using directly unix file descriptors and would make the code less maintainable.

Reply:
Can we add some options into Makefile to restrain this warning?

Reply:
I do not want to restrain this warning. It reminds us that we are using a potentially unstable error match. It's good to have it there